src/dpl/src/objective/detailed_abu.cxx

```diff
@@ -434,18 +434,26 @@ double DetailedABU::delta(const Journal& journal)
   // Compute changed bins and changed occupancy.
   ++abuChangedBinsCounter_;
   const auto& actions = journal.getActions();
-  for (const auto& action : actions) {
-    auto node = action.getNode();
-    updateBins(action.getNode(),
-               action.getOrigLeft().v + 0.5 * node->getWidth().v,
-               action.getOrigBottom().v + 0.5 * node->getHeight().v,
+  for (const auto& uaction : actions) {
+    if (uaction->typeId() != JournalActionTypeEnum::MOVE_CELL) {
+      continue;
+    }
```

Comment:
It feels odd that client code is going through the action list.   I need a better understanding of why this is part of the architecture (not clearly related to this PR but I'm noticing it now).